game_menu.py

```diff
@@ -30,6 +39,37 @@ def draw(self):
         for button in self.button_list:
             button.draw()
 
+    def update(self, delta_time):
+        for button in self.button_list:
+            # detect if the button is moused over
+            mouse_on = True
+            if self.game.mouse_x > button.center_x + button.width / 2:
+                mouse_on = False
+            elif self.game.mouse_x < button.center_x - button.width / 2:
+                mouse_on = False
+            elif self.game.mouse_y > button.center_y + button.height / 2:
+                mouse_on = False
+            elif self.game.mouse_y < button.center_y - button.height / 2:
+                mouse_on = False
```

Comment:
Replacing `mouse_on = False` with `continue` runs more efficiently and removes the need for the `if` statement on line 56